4.9/ga/ci-pipeline.md

```diff
@@ -14,6 +14,7 @@
   * [Step 3 - Create an OpenShift Project (namespace) to work in](#step3)
   * [Step 4 - Add the Kubeconfig secret](#step4)
   * [Step 5 - Import Red Hat Catalogs](#step5)
+* [Optional Step - If using an OpenShift on Power cluster](#anyuid-scc-ppc64e)
```

Comment:
Can you align the initial spaces with the rest of the bullet points?

Reply:
@Allda Done!